sqs/src/test/scala/akka/stream/alpakka/sqs/scaladsl/SqsSpec.scala

```diff
@@ -188,6 +188,26 @@ class SqsSpec extends FlatSpec with Matchers with DefaultTestContext {
     probe.cancel()
   }
 
+  it should "pull messages from a fifo queue following the same production order" taggedAs Integration in {
+    val queue = randomFifoQueueUrl()
+    implicit val awsSqsClient = sqsClient
+
+    for (i <- 0 until 10) yield {
```

Comment:
The `yield` doesn't do anything useful now.

Reply:
Oppss, sorry about that. I little bit of context switching and to many time of for comprehensions instead of for loops. Fixed!